api/app/hfi/hfi_request.py

```diff
@@ -0,0 +1,38 @@
+""" HFI request logic """
+
+from itertools import groupby
+import operator
+from typing import List
+from app.db.models.hfi_calc import PlanningWeatherStation
+from app.schemas.hfi_calc import HFIResultRequest, StationInfo
+
+
+def update_result_request(result_request: HFIResultRequest,
+                          latest_stations: List[PlanningWeatherStation]):
+    """ Returns copy of the result request with the latest planning weather stations """
```

Comment:
sorry, what do you mean by "latest" stations?

Reply:
`latest_persisted_stations`?

Reply:
`currently_persisted_stations`?

Reply:
I like `currently_persisted_stations` best